src/components/GenericWarningModal/GenericWarningModal.tsx

```diff
@@ -0,0 +1,46 @@
+import {
+  Text,
+  HStack,
+  Modal,
+  ModalBody,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+} from "@chakra-ui/react"
+import { ModalCloseButton } from "@opengovsg/design-system-react"
+import parse from "html-react-parser"
+
+interface ModalProps {
```

Comment:
what's our abstraction boundary here? doing this breaks the illusion that this is a modal and can be styled accordingly. if we aim to provide the user with some level of customization (in the future at least), we should aim to extend the according interface exported from chakra. (either `Modal/ModalBody` depending on the intention of this component).

**however**, the caveat here is that if the intention is to limit customizability then this is ok. just need clarification here

Reply:
As discussed offline, I think it makes sense to standardise the overall modal layout as much as possible - but we should allow for greater customisability of the modal content as well. The displayText has been converted into a JSX element and the parse has been removed in https://github.com/isomerpages/isomercms-frontend/pull/941/commits/eb973f7885ef6e088e30376fd92c04cfc156f085